docs/deploy/azure.md

```diff
@@ -19,11 +19,90 @@ next:  ["Import data into your installation", "../howto/import.html"]
 
 {% include_relative includes/prerequisites.md %}
 
+## Supported storage types
+
+lakeFS supports the following storage account types:
+
+1. Azure Blob Storage
+2. Azure Data Lake Storage Gen2 (HNS)
+
+Other Azure storage types have not been tested, and specifically Data Lake Storage Gen1 is not supported.  
+For more information see: [Introduction to Azure Storage](https://learn.microsoft.com/en-us/azure/storage/common/storage-introduction)
+
+## Authentication methods
+
+lakeFS supports two ways to authenticate with Azure:
+* **Storage account based authentication**
+* **Identity based authentication**
+
+### Storage Account Credentials
+
+Storage account credentials can be set directly in the lakeFS configuration using the following parameters:
+
+* `blockstore.azure.storage_account`
+* `blockstore.azure.storage_access_key`  
+
+Please note that using this authentication method limits lakeFS to the scope of the given storage account. Specifically, **the following operations will not work**:
+
+1. Import of data from different storage accounts
+2. Copy/Read/Write of data that was imported from a different storage account
+3. Create pre-signed URL for data that was imported from a different storage account
+
+### Identity Based Credentials
+
+lakeFS uses environment variables to determine credentials to use for authentication. the following authentication methods are supported:
+1. Managed Service Identity (MSI)
+2. Service Principal RBAC
+3. Azure CLI
```

Comment:
It this a tool to authenticate with or another type of authentication like MSI? I know that the Azure docs are confusing, I don't want to import that confusion here as well

Reply:
No tool - just different env vars for different methods of authentication